### ISSUE TITLE
Make PGP keyserver configurable

### ISSUE THREAD

**Issue by RichieB2B** [MISP/MISP#707]
The PGP keyserver pgp.mit.edu is hardcoded in app/Model/User.php app/webroot/js/ajaxification.js
While this server is a good choice as a default, this really should be configurable per MISP instance.

**Comment by iglocska** [comment]
Yeah this is something on my bucket list. It would be nice to have other options, but the problem is that the format in which keyservers return data is not uniform, meaning that we need specific parsers / scrapers for them. I think the best would be to simply include a list of servers in the long run.
